File: 10040.cpp
#include <iostream>
using namespace std;

int* funny;
int* vote;

int main() {
	int N, M; cin >> N >> M;

	funny = new int[N];
	vote = new int[N];
	for (int i = 0; i < N; i++) {
		cin >> funny[i];
		vote[i] = 0;
	}

	for (int i = 0; i < M; i++) {
		int B; cin >> B;

		for (int j = 0; j < N; j++) {
			if (funny[j] <= B) {
				vote[j]++;
				break;
			}
		}
	}

	int max = vote[0], max_index = 1;
	for (int i = 1; i < N; i++) {
		if (vote[i] > max) {
			max = vote[i];
			max_index = i + 1;
		}
	}

	cout << max_index << endl;

	delete[] funny;
	delete[] vote;
}

File: 10040.py
N, M = map(int, input().split())

funny = []
for i in range(N):
    funny.append(int(input()))
vote = [0] * N

for i in range(M):
    B = int(input())

    for j in range(N):
        if funny[j] <= B:
            vote[j] += 1
            break

max = vote[0]; max_index = 1
for i in range(1, N):
    if vote[i] > max:
        max = vote[i]
        max_index = i + 1

print(max_index)
